Example/Core/Tests/FIRAppTest.m

```diff
@@ -620,7 +622,7 @@ - (void)testGlobalDataCollectionNoDiagnosticsSent {
   // Add an observer for the diagnostics notification. Currently no object is sent, but in the
   // future that could change so leave it as OCMOCK_ANY.
   [self.notificationCenter addMockObserver:self.observerMock
-                                      name:kFIRAppDiagnosticsNotification
```

Comment:
Any particular reason we are renaming the constant?

Reply:
It was for consistency, but I'm actually removing the notification based structure in favor of a component based structure.